packages/xo-server/src/api/pool.js

```diff
@@ -111,6 +112,22 @@ installPatches.description = 'Install patches on hosts'
 
 // -------------------------------------------------------------------
 
+export async function rollingUpdate({ pool }) {
+  await this.getXapi(pool).rollingPoolUpdate({
+    restartHostTimeout: parseDuration(this._config.patches.rollingPoolUpdate.restartHostTimeout),
```

Comment:
Not sure if there's a better way to do it.

Reply:
You can simply use it in `mixins/patching.js`.

Reply:
Ah Ok, i see, it's not a class.